zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/DecisionDefinitionController.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller;
+
+import io.camunda.service.DecisionDefinitionServices;
+import io.camunda.service.search.query.DecisionDefinitionQuery;
+import io.camunda.zeebe.gateway.protocol.rest.DecisionDefinitionSearchQueryRequest;
+import io.camunda.zeebe.gateway.protocol.rest.DecisionDefinitionSearchQueryResponse;
+import io.camunda.zeebe.gateway.rest.RestErrorMapper;
+import io.camunda.zeebe.gateway.rest.SearchQueryRequestMapper;
+import io.camunda.zeebe.gateway.rest.SearchQueryResponseMapper;
+import io.camunda.zeebe.gateway.rest.TenantAttributeHolder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@ZeebeRestController
+public class DecisionDefinitionController {
+
+  @Autowired private DecisionDefinitionServices decisionDefinitionServices;
+
+  @PostMapping(
+      path = "/decision-definitions/search",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<DecisionDefinitionSearchQueryResponse> searchDecisionDefinitions(
+      @RequestBody final DecisionDefinitionSearchQueryRequest query) {
```

Comment:
🔧 With this, the message body is required. That forces users to at least send an empty body. If we can work with an "undefined" query as well (and the tests imply that we can), I'd recommend setting the body to `required = false` and also testing that this works as expected.

Reply:
it makes sense, I will update both `searchDecisionDefinitions` and `searchProcessInstances`

Reply:
✅